cpp/src/arrow/compute/kernels/CMakeLists.txt

```diff
@@ -93,9 +90,12 @@ add_arrow_benchmark(vector_selection_benchmark PREFIX "arrow-compute")
 
 # Aggregates
 
-add_arrow_compute_test(aggregate_test
-                       SOURCES
-                       aggregate_test.cc
-                       hash_aggregate_test.cc
+add_arrow_compute_test(aggregate_test SOURCES aggregate_test.cc
+                       #                       hash_aggregate_test.cc
                        test_util.cc)
-add_arrow_benchmark(aggregate_benchmark PREFIX "arrow-compute")
+# add_arrow_benchmark(aggregate_benchmark PREFIX "arrow-compute")
```

Comment:
Is it WIP?

Reply:
moved in acero since it uses acero extensively